lib/services/access_request.go

```diff
@@ -1132,8 +1169,14 @@ func (m *RequestValidator) Validate(ctx context.Context, req types.AccessRequest
 		return trace.BadParameter("request validator configured for different user (this is a bug)")
 	}
 
-	if m.requireReason && req.GetRequestReason() == "" {
-		return trace.BadParameter("request reason must be specified (required by static role configuration)")
+	if len(req.GetRequestReason()) == 0 {
```

Comment:
I think we should do strings.TrimSpace on this to prevent basic cheating like a bunch of whitespaces.

Reply:
https://github.com/gravitational/teleport/pull/49124/commits/b9389c45d7820013389eba731191bb1ea99dc6ba